BlueBrain/CoreNeuron
# Print memory use from model_size

**Description**

Small patch cleaning up `model_size` and using it to print memory requirements at the end of `nrn_setup` the print is only done when coreneuron is _not_ run in quiet mode.

**How to test this?**

Just a normal run should show the print

```bash
cmake ..
make -j8
nrnivmodl mod
./bin/nrnivmodl-core mod
./x86_64/special script.py
./x86_64/special-core --tstop=10 --datpath=coredat
```

**Use certain branches for the SimulationStack CI**

CI_BRANCHES:NEURON_BRANCH=master,
